pam/pam.go

```diff
@@ -430,12 +406,8 @@ func formChallenge(client authd.PAMClient, sessionID, encryptionKey string, uiLa
 				SessionId:          sessionID,
 				AuthenticationData: `{"wait": "true"}`,
 			})
-
-			// No more processing if entry has been filed.
-			select {
-			case <-waitCtx.Done():
```

Comment:
are you sure we can’t remove waitCtx completely?

Reply:
We could rename it to something like `firstCallCtx` to be more clear about its meaning...